src/status_im/ui/screens/browser/views.cljs

```diff
@@ -64,6 +63,13 @@
    [react/view styles/web-view-loading
     [components/activity-indicator {:animating true}]]))
 
+(defn on-bridge-message [message browser]
+  (let [{:strs [type navState]} (js->clj (.parse js/JSON message))
+        {:strs [url title]} navState]
+    (when (= type "navStateChange")
+      (when (not= "about:blank" url)
```

Comment:
might be `and` instead of the second `when`

Reply:
there will be more types, will be condition